breathecode/feedback/tests/tasks/tests_generate_user_cohort_survey_answers.py

```diff
@@ -77,53 +78,57 @@ def test_when_teacher_is_assigned(self):
 
             generate_user_cohort_survey_answers(model.user, model.survey, status="OPENED")
 
+            answers = [
+                {
+                    "title": f"How has been your experience studying {model.cohort.name} so far?",
+                    "lowest": "not good",
+                    "highest": "very good",
+                    "cohort_id": n + 1,
+                    "academy_id": n + 1,
+                    "token_id": None,
+                },
+                {
+                    "title": f"How has been your experience with your mentor {model.user.first_name} {model.user.last_name} so far?",
+                    "lang": "en",
+                    "mentor_id": n + 1,
+                    "lowest": "not good",
+                    "mentorship_session_id": None,
+                    "score": None,
+                    "sent_at": None,
+                    "status": "OPENED",
+                    "highest": "very good",
+                    "cohort_id": n + 1,
+                    "academy_id": n + 1,
+                },
+                {
+                    "title": f"How likely are you to recommend {model.academy.name} to your friends " "and family?",
+                    "lowest": "not likely",
+                    "highest": "very likely",
+                    "cohort_id": None,
+                    "academy_id": n + 1,
+                },
+                {
+                    "title": f"How has been your experience with the platform and content?",
+                    "lowest": "not good",
+                    "highest": "very good",
+                    "cohort_id": None,
+                    "academy_id": None,
+                    "question_by_slug": "PLATFORM",
```

Comment:
If a cohort is set and is available as saas or an academy is not available as saas, only an answer should be generated related to `"question_by_slug": "PLATFORM",`

Reply:
So, I mean a case must be 

```py
answers = [
                {
                    "title": f"How has been your experience with the platform and content?",
                    "lowest": "not good",
                    "highest": "very good",
                    "cohort_id": None,
                    "academy_id": None,
                    "question_by_slug": "PLATFORM",
                 },
]
```